inputFiles/thermoPoromechanicsFractures/plotsolution_thermoPoroelastic.py

```diff
@@ -0,0 +1,102 @@
+import matplotlib
+import matplotlib.pyplot as plt
+import numpy as np
+import h5py
+import xml.etree.ElementTree as ElementTree
+from mpmath import *
+import math
+
+
+def main():
+    # File paths
+    hdf5FilePathTemperature = "/usr/WS2/cusini1/geosx/geosx_dev/GEOSX/build-quartz-gcc@12-debug/Output/temperature_history.hdf5"
+
+    
+    # Read simulation output from HDF5 file
+    hf = h5py.File(hdf5FilePathTemperature, 'r')
+    timeTemperature = hf.get('temperature Time')
+    timeTemperature = np.array(timeTemperature)
+    centerTemperature = hf.get('temperature elementCenter')
+    centerTemperature = np.array(centerTemperature)
+    xcord_elm = centerTemperature[0,:,0]
+    ycord_elm = centerTemperature[0,:,1]
+    zcord_elm = centerTemperature[0,:,2]
+    temperature = hf.get('temperature')
+    temperature = np.array(temperature)
+    print(ycord_elm)
+
+    hf = h5py.File("/usr/WS2/cusini1/geosx/geosx_dev/GEOSX/build-quartz-gcc@12-debug/Output/pressure_history.hdf5", 'r')
+    pressure = np.array( hf.get('pressure') )
+    
+    t_index = [0, 1, 2, 9]
+    tstar = [0, 100, 200, 900]     
+    numFiles=len(tstar)
+    # Extract Curve
+    midY = 11.0  
+    xlist = []
+    pplist = []
+    temperaturelist = []
+    for tt in range(len(t_index)):
+        xtemp = []
+        ptemp = []
+        ttemp = []
+        for i in range(0,len(zcord_elm)):
+            if abs(ycord_elm[i] - midY) < 0.01:
+               xtemp.append(xcord_elm[i])
+               ptemp.append(pressure[tt,i]/1.0e5)
```

Comment:
```suggestion
               ptemp.append(pressure[tt,i]/1.0e6)
```

Reply:
here you want to convert from Pa to MPa?

Reply:
I d say to bar so the `1.e5` is correct.